package.json

```diff
@@ -108,6 +108,6 @@
     "typescript": "^4.2.4"
   },
   "dependencies": {
-    "cross-fetch": "^3.1.4"
```

Comment:
Removing cross fetch is intentional? AFAIK its used by SSR

Reply:
nope, it is also required for tests, removal was made by mistake.